site/_posts/2020-06-03-user-registration-and-sign-in-with-laravel.md

```diff
@@ -218,7 +248,7 @@ The login form is very similar to the registration form created in the previous
 @endsection
 ```
 
-Finally, to process user logins, you'll need to create another single action controller. First, run the artisan command:
+To process user logins, you'll need to create another single action controller. First, run the artisan command:
```

Comment:
Is artisan a cli tool? If so, \`artisan\` is better.

Reply:
`php artisan` gives you a subset of Laravel commands - will adjust markdown 👍🏻